code/src/components/Details.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import '../details.css'
+
+const Details = () => {
+  const [details, setDetails] = useState({});
+  const navigate = useNavigate();
+  // eslint-disable-next-line camelcase
+  const { movieId } = useParams();
+
+  const onBackButtonClick = () => {
+    navigate(-1);
+  }
+  useEffect(() => {
+    // eslint-disable-next-line camelcase
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=c77f0f257b1db8c5cdfe593182e32f79&language=en-US`)
+      .then((res) => res.json())
+      .then((data) => {
+        setDetails(data)
+      })
+  }, [movieId])
+
+  return (
+    <div className="detailsWrapper">
+      <div className="btnWrapper">
+        <button type="button" className="backBtn" onClick={onBackButtonClick}><FontAwesomeIcon icon={faCircleChevronLeft} /><span className="backBtnName">Movies</span></button>
+      </div>
+      <div className="background">
+        <img alt="img" className="backdropImg" src={`https://image.tmdb.org/t/p/w1280${details.backdrop_path}`} />
+        <div className="foreground">
+          <img alt="img" className="detailsPosterImg" src={`https://image.tmdb.org/t/p/w342${details.poster_path}`} />
+          <div className="movieDetails">
+            <h3><span className="title">{details.title}</span><span className="rating"><span className="starIcon">⭐️</span>{Math.round(details.vote_average * 10) / 10}</span></h3>
+            <p className="overview">{details.overview}</p>
+          </div>
```

Comment:
Maybe some of the classnames can be even more descriptive, related to what role they have in the whole site/the component they belong to? Like "singleMovieDetailsBackground" or something like that. I just got feedback from my mentor not to fear long names, it just makes it easier for others to understand our code!